src/fragments/MessageListItem.tsx

```diff
@@ -66,6 +75,18 @@ const MessageListItem: React.FC<MessageListItemProps> = ({
     }
   }, []);
 
+  useEffect(() => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
The `any` cast here isn't necessary. Use the `isTextMessage` type helper function to check if the message is a  `TextMessage` and then access the `mentions` property.